x-pack/test/api_integration/apis/security/security_basic.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ loadTestFile }: FtrProviderContext) {
+  describe('security (basic license)', function() {
+    this.tags('ciGroup6');
+
+    // Updates here should be mirrored in `./index.js` if tests
+    // should also run under a trial/platinum license.
+
+    loadTestFile(require.resolve('./basic_login'));
+    loadTestFile(require.resolve('./builtin_es_privileges'));
+    loadTestFile(require.resolve('./change_password'));
+    loadTestFile(require.resolve('./index_fields'));
+    loadTestFile(require.resolve('./roles'));
+    loadTestFile(require.resolve('./privileges_basic'));
```

Comment:
Looking at this, I wonder if it would be sufficient just to test `privileges_basic` in this scenario. Of course the current approach gives us "more complete" test coverage, but in all the other test files we're just retesting functionality that won't change based on license type.

Reply:
We were previously running all of these tests for both licenses via https://github.com/elastic/kibana/pull/59750/files#diff-d2e8e846459935da35e52b32d2fbde0fL17. Before this PR, the existing tests could all be run on both license types, with identical expected results.

Now, we have a distinction in output between basic and trial licenses, but I still think it makes sense to duplicate this testing for the rest of the existing tests. This was done in our original initiative to make security features available for free under our basic license.